docs/source/release_notes.rst

```diff
@@ -2,21 +2,25 @@
 
 Release Notes
 -------------
+.. **Future Release**
+..     * Enhancements
+..     * Fixes
+..     * Changes
+..     * Documentation Changes
+..     * Testing Changes
```

Comment:
The styling for this commented out section should match what is shown in `release.md`:

```
.. Future Release
  ==============
    * Enhancements
    * Fixes
    * Changes
    * Documentation Changes
    * Testing Changes

.. Thanks to the following people for contributing to this release:
```